pkg/runtest/run_test.go

```diff
@@ -414,7 +415,13 @@ func TestParsing(t *testing.T) {
 			t.Run(fmt.Sprintf("%v/%v", target.OS, target.Arch), func(t *testing.T) {
 				t.Parallel()
 				for _, file := range files {
-					p, requires, _, err := parseProg(target, dir, file)
+					// syz_mount_image tests are very large and this test takes too long.
+					// syz-imagegen that generates does some of this testing (Deserialize/SerializeForExec).
+					if strings.HasPrefix(file, "syz_read_part_table") ||
```

Comment:
Maybe split `parseProg` into `parseProg` and `parseRequires`? It will help us avoid introducing more hard-coded call names.

Reply:
Done. PTAL.